src/content/guides/getting-started.md

```diff
@@ -99,7 +102,7 @@ __project__
 To bundle the `lodash` dependency with `index.js`, we'll need to install the library locally...
 
 ``` bash
-npm install --save lodash
+npm install --save-dev lodash
```

Comment:
This should be `npm install --save lodash` or bundling won't work.

Reply:
Are you sure about that? It works fine in devDependencies (and it should).

This will only be an issue if you have an app that runs on Node and has a dependency on the server. Client-side bundlers should use devDependencies usually.

I just tested it and it works fine either way.

Reply:
By definition, `dependencies` are needed at run time, doesn't matter if you're bundling for Node or the browser. 

In contrast, `devDependencies` are needed only while developing, again indifferent from where the bundled output is going to run.

Here `lodash` will actually run in the application and not used only during development.

Reference: https://stackoverflow.com/questions/18875674/whats-the-difference-between-dependencies-devdependencies-and-peerdependencies

Reply:
I'm fairly sure that in this case lodash gets bundled by webpack, in fact, it would make no sense to have webpack installed as a devDependency but lodash as a regular dependency.

It's a trivial thing, but I'm a proponent of keeping things consistent and I don't see a reason to treat lodash differently in this case.

If you want you can give it a shot yourself and install it as a dev dependency, webpack will still bundle it just fine. It's not being loaded externally.

Can someone else maybe pitch in on this?
It seems to be a growing trend where packages always get installed with `--save` where it's not required (in most cases).

Reply:
@TheLarkInn @sokra word on this?

Reply:
Both works for `webpack` since it doesn't care where the contents in `node_modules` are from :). IHMO something like `lodash` is a `dependency` of your `site/spa/...` once it's also included into 'production' code, so `npm i lodash` is the 'current' usage here

Reply:
This is a highly confusing topic - but it's all on the perspective of what to expect when another package lists your webpack build as a dependency.

If lodash is bundled by webpack into the dist bundle, it's not needed by the upstream package. So it should be in `devDependencies`.

Reply:
In this specific example `lodash` should be in `dependencies`, it is a library needed at runtime, that is bundled my webpack.
In real world examples/applications you never bundle eslint or prettier into your bundles, that's why those are installed as `devDependencies`

Reply:
When you say:
> To bundle the lodash dependency with index.js, we'll need to install the library locally...

It means that it is necessary at runtime, bundled.

Meanwhile
> npm install --save-dev lodash

says: `lodash` is used for development purposes, similar to Flow, Prettier or ESLint, in this is not the case.

And finally:
> and then import it in our script...

Our application script.

Reply:
Cool, this all makes sense!

Thanks so much for enlightening me on this subject. It has always bothered me and didn't make a lot of sense and I think a lot of beginners feel the same way.

I'll make sure to include a piece of this in the Guides as well!

🎉